config/fxdata/magic.cfg

```diff
@@ -944,9 +944,6 @@ SpellEffect = 0
 Speed = 95
 BaseExperienceGain = 256
 DestroyOnHit = 0
-BounceAngle = 0
-BaseExperienceGain = 256
-DestroyOnHit = 1
```

Comment:
I'm not sure if it's supposed to be set to 1 or 0, I assume it's 0 cause it has NO_HIT?

Reply:
I checked and you deleted the correct ones.